build/Microsoft.DotNet.Cli.Publish.targets

```diff
@@ -7,7 +7,7 @@
 
   <!-- PUBLISH_TO_AZURE_BLOB env variable set by CI -->
   <Target Name="Publish"
-          Condition=" '$(PUBLISH_TO_AZURE_BLOB)' != '' "
+          Condition=" '$(PUBLISH_TO_AZURE_BLOB)' == 'true' "
```

Comment:
deliberate? bool in msubild is not used consistently

Reply:
This was indeed a deliberate change.